src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -58,10 +58,22 @@ public function __toString()
     /**
      * Returns the headers.
      *
+     * @param string|null $key get all based on key
```

Comment:
s/get all based on key/The name of the headers to return or null to get them all